base/hashing.jl

```diff
@@ -17,89 +22,62 @@ The hash value may change when a new Julia process is started.
 
 ```jldoctest
 julia> a = hash(10)
-0x95ea2955abd45275
+0x64abb6a0b5357846
```

Comment:
Maybe add a filter to the doctest to instead test that it returns something matching the regex `0x[0-9a-f]{16}`?